src/rpdk/core/test.py

```diff
@@ -441,6 +443,16 @@ def setup_subparser(subparsers, parents):
         "of SAM",
     )
 
+    parser.add_argument(
+        "--source-account",
+        help="Source Account key used for Assume Role to Run Contract Tests",
+    )
+
+    parser.add_argument(
+        "--source-arn",
+        help="Source Type Version Arn key used for Assume Role to Run Contract Tests",
+    )
```

Comment:
do you need to add basic validation? what are the implications if incorrect entities are provided